### DISCUSSION TITLE
Support block height for reminders

### DISCUSSION THREAD

**Discussion by ekzyis** [stackernews/stacker.news#3029] [Ideas]
### Describe the problem you're trying to solve

Not sure there's a problem that this would solve but ...

### Describe the solution you'd like

... at least two stackers think it would be cool if our `@remindme` bot could support relative or absolute block height.

For example: `@remindme in 3 blocks` or `@remindme at block 888888`

### Describe alternatives you've considered

n/a

### Additional context

https://stacker.news/items/789455?commentId=789482

**Comment by huumn** [comment]
We could also do it for deletes.

I guess it might be useful for onchain predictions
